www/css/transition.css

```diff
@@ -0,0 +1,53 @@
+.slideOut_R {
+    transform: translateX(0%);
+    animation: moveOutR 0.3s linear forwards;
+}
+.slideIn_R {
+    transform: translateX(-100%);
+    animation: moveInR 0.3s linear forwards;
+}
+@keyframes moveOutR {
+    0% {
+        transform : translateX(0%);
+

```

Comment:
@vasusharma7 It's very minor, but remove this redundant blank line (line 12), and similar ones below (38, 48).

Reply:
OkI will do so